app/controllers/idv/in_person/verify_info_controller.rb

```diff
@@ -74,7 +74,8 @@ def prev_url
       end
 
       def pii
-        user_session.dig('idv/in_person', :pii_from_user).merge(ssn: idv_session.ssn)
+        pii_from_user = user_session.dig('idv/in_person', :pii_from_user) || {}
```

Comment:
How does IPP end up in a state where `pii` is nil at this point? It should already exist right?

Reply:
I was wondering the same! We observed errors in NewRelic that traced back to pii being nil here. I couldn't reproduce it manually. Not sure if this indicates something bigger worth investigating? Perhaps handling the error here isn't sufficient? Open to discussion!

Reply:
For posterity, discussion was continued internally [here](https://gsa-tts.slack.com/archives/C0NGESUN5/p1723238948807369?thread_ts=1719342027.961919&cid=C0NGESUN5)

Reply:
Revisited this, and after investigating the logs I found that a user could cancel their in person proofing flow at the verify info page, exit login.gov, then hit the back button to re-enter login.gov, and hit "keep going" to return to the verify info page, where they would no longer have PII in their session since it was cleared on exit. 

So, I've added a before_action to redirect if there is no IPP session data present. This follows the behavior on the remote flow side, where the verify info controller also checks that that the step is valid and otherwise redirects.